<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/diffimg.qrc">
    <normaloff>:/diffimg/diffimg.png</normaloff>:/diffimg/diffimg.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuForm">
    <property name="title">
     <string>Form</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuForm"/>
   <addaction name="menuView"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoadFilelist"/>
   <addaction name="actionSnapshot"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionFitToWindow"/>
   <addaction name="actionFullResolution"/>
   <addaction name="actionPrev"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionColumn">
   <property name="text">
    <string>Column</string>
   </property>
  </action>
  <action name="actionRow">
   <property name="text">
    <string>Row</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/icon_quit.png</normaloff>:/diffimg/icon_quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/icon_open.png</normaloff>:/diffimg/icon_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFullResolution">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/icon_mag1.png</normaloff>:/diffimg/icon_mag1.png</iconset>
   </property>
   <property name="text">
    <string>Full resolution</string>
   </property>
  </action>
  <action name="actionFitToWindow">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/icon_magfit.png</normaloff>:/diffimg/icon_magfit.png</iconset>
   </property>
   <property name="text">
    <string>Fit to window</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/next.png</normaloff>:/diffimg/next.png</iconset>
   </property>
   <property name="text">
    <string>actionNext</string>
   </property>
   <property name="toolTip">
    <string>Next image</string>
   </property>
   <property name="statusTip">
    <string>Next image</string>
   </property>
   <property name="whatsThis">
    <string>Next image</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/prev.png</normaloff>:/diffimg/prev.png</iconset>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="toolTip">
    <string>Previous image</string>
   </property>
   <property name="statusTip">
    <string>Previous image</string>
   </property>
   <property name="whatsThis">
    <string>Previous image</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/save.png</normaloff>:/diffimg/save.png</iconset>
   </property>
   <property name="text">
    <string>Save difference image</string>
   </property>
   <property name="toolTip">
    <string>Save difference image</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/refresh.png</normaloff>:/diffimg/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionLoadFilelist">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/perchannel.png</normaloff>:/diffimg/perchannel.png</iconset>
   </property>
   <property name="text">
    <string>LoadFilelist</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="icon">
    <iconset resource="../res/diffimg.qrc">
     <normaloff>:/diffimg/snapshot.png</normaloff>:/diffimg/snapshot.png</iconset>
   </property>
   <property name="text">
    <string>Snapshot</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/diffimg.qrc"/>
 </resources>
 <connections/>
</ui>
